client/README.md

```diff
@@ -1,70 +1,25 @@
-# Getting Started with Create React App
+# NextRound Client
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Routes
 
-## Available Scripts
+`/`: Landing page
 
-In the project directory, you can run:
+`/home`: Main home page for the current cycle
 
-### `npm start`
+`/home/:id`: Main home page for any cycle with id of `id`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+`/metrics`: Metrics dashboard for the current cycle
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+`/metrics/:id`: Metrics dashboard for any cycle with id of `id`
 
-### `npm test`
+`/social`: Social media page
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+`/settings`: Settings page for the current user
 
-### `npm run build`
+`/job/:jobID`: Single job page for the job with id of `jobID` within the current cycle
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+`/job/:jobID/cycleID`: Single job page for the job with id of `jobID` within the cycle with id of `cycleID`
```

Comment:
Small comment but shouldn't it be `/job/:jobID/:cycleID" ie there should be a colon mark between the last forward slash and cycleID